tests/test-definition.yaml

```diff
@@ -20,7 +21,7 @@ dimensions:
       - 3.9.2
   - name: zookeeper-latest
     values:
-      - 3.9.2
+      - 3.9.3
```

Comment:
if we change "zookeeper-latest", shouldn't we include at version under "zookeeper"? If not we will short-circuit the chaos-monkey test, which only runs if zookeeper=zookeeper-latest (see https://github.com/stackabletech/hdfs-operator/blob/main/tests/templates/kuttl/smoke/60-unleash-the-chaosmonkey.yaml.j2#L5)

Reply:
ah, have just seen @Techassi's comment above :)

Reply:
https://github.com/stackabletech/hdfs-operator/pull/656/commits/3fed11693f7c56f5f7be1e9c7b03a30cf9194c55